components/feature/top-sites/src/main/java/mozilla/components/feature/top/sites/PinnedSiteStorage.kt

```diff
@@ -16,6 +17,9 @@ import mozilla.components.feature.top.sites.db.toPinnedSite
  */
 class PinnedSiteStorage(context: Context) {
 
+    @VisibleForTesting
+    internal var currentTimeMillis: () -> Long = { System.currentTimeMillis() }
```

Comment:
I use it to inject a constant time from the tests (`add al default sites` and `add pinned site`) so that I can use `verify` without much effort. What would be a good alternative?

Reply:
I think we can just check for anyLong() as an alternative.

Reply:
I tried that but I think it doesn't work because the long is wrapped in an object. I think this works:
`verify(obj).method(eq("foo"), anyLong())`
But this doesn't:
`verify(obj).method(SomeClass(eq("foo"), anyLong()))`
And our case matches the latter. But I don't really have a lot of experience in this area so please correct me if I'm missing something. The real problem I think we're having is that there is no consistent way to mock time used throughout the project. Code that does straight calls to `System.currentTimeMillis()` is just not very unit-testable. Perhaps we should make an issue about this?